db/controllers/app.controller.js

```diff
@@ -1,4 +1,19 @@
-const { fetchTopics } = require("../models/app.model");
+const { fetchTopics, fetchArticles } = require("../models/app.model");
+exports.getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((results) => {
+      res.status(200).send(results);
```

Comment:
We should send the data back on its own key which we can do by wrapping the argument in end with curly braces